pom.xml

```diff
@@ -18,7 +18,7 @@
 
     <groupId>uk.co.caprica</groupId>
     <artifactId>vlcj-info</artifactId>
-    <version>2.0.4-SNAPSHOT</version>
```

Comment:
There is no need to bump this version number, it's already a snapshot.

Your change would become part of 2.0.4 final when released.

Reply:
Will do